entity/c/c_api.c

```diff
@@ -0,0 +1,197 @@
+#include "c_api.h"
+
+extern int sent_seq_num;
+extern unsigned char entity_client_state;
+extern unsigned char entity_server_state;
+extern long int st_time;
+
+config_t *load_config(char *path)
+{
+    return load_config_t(path);
+}
+
+session_key_t *get_session_key(config_t *config_info)
+{
+    if (strncmp((const char *)config_info->network_protocol, "TCP", 3) == 0)
```

Comment:
Why can't you use just strcmp? Isn't `config_info->network_protocol` null-terminated?